review/fileToReview.js

```diff
@@ -63,11 +80,11 @@ async function fetchTransactions(fromDate, authorization, jws = null, id, page,
     } else throw new Error();
 
     return [];
-	} catch (err) {
-		throw new CustomError({
```

Comment:
* CustomError is not a native class, it has to be defined to exist, it is not the case in here